src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -508,14 +508,15 @@ private static int CheckUserAndGroupIds(string userId, string groupId, string gr
             Assert.Equal(groupId, getgid().ToString());
             Assert.Equal(groupId, getegid().ToString());
 
+            var expectedGroups = new HashSet<uint>(groupIdsJoined.Split(',').Select(s => uint.Parse(s)));
+
             if (bool.Parse(checkGroupsExact))
             {
-                Assert.Equal(groupIdsJoined, string.Join(",", GetGroups()));
+                Assert.Equal(expectedGroups, GetGroups());
```

Comment:
Why isn't sorting needed in this case? Will there only ever be one id?

Reply:
If you pass Assert.Equal a set, then it does a hashset comparison. Should I add a comment about this?

Reply:
Interesting; hadn't realized that. No comment necessary.